webdriver/tests/classic/perform_actions/pointer_mouse.py

```diff
@@ -204,6 +204,53 @@ def test_move_to_position_in_viewport(
     assert len(events) == event_count
 
 
+@pytest.mark.parametrize("origin", ["viewport", "pointer", "element"])
+def test_move_to_origin_position_within_frame(
+    session, iframe, inline, mouse_chain, origin
+):
+    session.url = inline(
+        iframe(
+            """
+        <input>
+        <script>
+            "use strict;"
+
+            var allEvents = { events: [] };
+            window.addEventListener("mousemove", e => {
+                allEvents.events.push([
+                    e.clientX,
+                    e.clientY,
+                ]);
+            });
+        </script>
+    """
+        )
+    )
+
+    frame = session.find.css("iframe", all=False)
+    session.switch_frame(frame)
+
+    elem = session.find.css("input", all=False)
+    elem_center_point = list(center_point(elem))
```

Comment:
question: do we really need to convert to a list here? We have a tuple and we never have to update it, so it should be good as is?

Reply:
No, we don't. Thank you for catching that. It's actually a relict from before a slight refactoring when I was comparing those values with an expected event build up as usual via a list. I'll remove.